spec/forms/search_spec.rb

```diff
@@ -44,6 +44,17 @@
         end
       end
 
+      context 'when an escort has been cancelled for a given prison number' do
+        let(:prison_number) { 'A1234BC' }
+        let(:detainee) { create(:detainee, prison_number: prison_number) }
+        let!(:escort) { create(:escort, prison_number: prison_number, detainee: detainee, cancelled_at: Time.now) }
```

Comment:
Can we please avoid using time values that are not in UTC within tests? We had issues before with this because the CI environment might be configured in a different timezone.

Also I would recommend this to be moved into a **trait** within the _escort_ factory so that can be re-used and it's an escort in a valid cancelled state (atm, has no associated user or reason)